independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/CreateProjectHelper.java

```diff
@@ -59,6 +77,9 @@ public static ExtensionCatalog completeCatalog(ExtensionCatalog catalog, Collect
         for (String extArg : extensions) {
             if (isFullArtifactCoords(extArg)) {
                 var coords = ArtifactCoords.fromString(extArg.trim());
+                if (coords.getVersion() == null) {
```

Comment:
Looks like `isFullArtifactCoords(s)` isn't doing what it should.

Reply:
If we remove this `if` will we get some test failures? I'm curious about when this happens.

Reply:
Ah no, I should remove this, it's was a wrong attempt..

Reply:
Good catch @aloubyansky